### PR TITLE
feat:[BMS-4] add eslint and prettierrc file

### PR DESCRIPTION
**Pull request by yashSP18**

### CODE REVIEW

File: eslint.config.mjs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        ...globals.node, // For Node.js global variables
+      },
+    },
+    rules: {
+      "no-unused-vars": "warn", // Warn on unused variables
+      "no-console": "warn", // Warn on console.log usage
+      eqeqeq: "error", // Require === and !==
+      semi: ["error", "always"], // Enforce semicolons at the end of statements
+      //"quotes": ["error", "double"], // Use single quotes
+      //"indent": ["error", 2],      // Enforce 2-space indentation
```
Comment (dhruvildave007) [inline eslint.config.mjs]: remove dead code if you are not using this linters

File: config/config.json [hunk 2]
```diff
@@ -1,23 +0,0 @@
...
-    "dialect": "mysql"
-  },
-  "production": {
-    "username": "root",
-    "password": null,
-    "database": "database_production",
-    "host": "127.0.0.1",
-    "dialect": "mysql"
-  }
-}
```
Comment (dhruvildave007) [inline config/config.json]: why have you deleted this

**Review by dhruvildave007 — CHANGES_REQUESTED** [review]

File: config/config.json (hunk 2 above)
Comment (dhruvildave007) [reply config/config.json]: have you created a different file for this?

Comment (yashSP18) [reply config/config.json]: because this come as extra in root folder I want only in src folder

Comment (yashSP18) [reply config/config.json]: yes that file is already in my src folder which is in previous pr and already merged

**Pull request merged**